components/src/core/drawer/drawer.component.ts

```diff
@@ -71,7 +71,7 @@ export class DrawerComponent extends StateListener implements OnInit, OnChanges
 
     // Close drawer on selection if drawer is only temporarily opened.
     // Expanding nested navitems when temporarily opened will not close the drawer.
-    private listenForDrawerSelection(): void {
+    listenForDrawerSelection(): void {
```

Comment:
same question as above about private->public

Reply:
I'm not sure why this method was marked as private in the first place so I removed this modifier.  A lot of our components could potentially have some private methods on it but our default for now has been to make everything public, so I just kept with that approach for now.